milton-processor/scribe.ts

```diff
@@ -46,13 +47,18 @@ app.post("/simplify", async (req, res) => {
 
   if (articleCached && !refreshCache) {
     console.log(`Found cached version of ${url}`);
-    manuscript = await cache.fetch(url);
+    manuscript = await cache.fetchManuscript(url);
     res.send(manuscript.data);
   } else {
     console.log(`Fetching contents of ${url}`);
-    readability.fetchArticle(url).then((response) => {
+    fetcher.fetchArticle(url).then((response) => {
       console.log(`Saving contents of ${url}`);
-        const po: readability.ParsedOutput = readability.parse(response, url);
+
+      if (response.isTextual()) {
+        // save raw contents as text and perform readability parsing
+        cache.saveRawContents(url, response.body, response.contentType);
+
+        const po: readability.ParsedOutput = readability.parse(response.body, url);
```

Comment:
It might make sense to only process `text/html` pages through readability

Reply:
fixed